frontend/src/styles/globals.css

```diff
@@ -24,4 +24,5 @@
   --font-size-lg: 32px;
   --font-size-xl: 40px;
   --main-container-spacing: 20px;
+  --box-shadow-purple: 0 4px 5px rgba(159, 24, 217, 50%);
```

Comment:
I was curious if you could use say `  --box-shadow-purple: 0 4px 5px rgba(var(--color-purple), 50%);`

But that's only possible if you would have ` --color-purple: 159, 24, 217;`

Which would break the rest. Just a curiosity, thought you'd like [reading](https://stackoverflow.com/questions/40010597/how-do-i-apply-opacity-to-a-css-color-variable) about it.